shared/constants/platform.desktop.js

```diff
@@ -140,7 +140,7 @@ function logFileName(): string {
     case 'darwin':
       return `${logDir()}/${envedPathOSX[runMode]}.app.log`
     case 'linux':
-      return '' // Linux is '' because we can redirect stdout
```

Comment:
This has been around for a really long time, right? Did something change recently that caused the logs not to get written properly? Or was this stuff always broken on Linux?

Reply:
Yeap, been around for a while. Now that I think about it, I've seen the 'context deadline exceeded' error before when doing `log send` on Linux, so it seems to me that this was just always broken. :scream:

Reply:
It shouldn't be broken, or at least not in the systemd case: https://github.com/keybase/client/blob/6e95ab068a0144cf9f9eaad843d9cb322493702d/packaging/linux/systemd/keybase.gui.service#L8

Reply:
Hmm, yeah looks like I misdiagnosed the original problem. The file does exist and is being upload -- the problem is that the UI log lines aren't being sent to stdout.